generator/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ add_library(
     internal/stub_factory_generator.cc
     internal/stub_factory_generator.h
     internal/stub_generator.cc
-    internal/stub_generator.h)
+    internal/stub_generator.h internal/sample_generator.cc internal/sample_generator.h)
```

Comment:
nit: this doesn't seem very alphabetical

Reply:
Fixed, I will take a look a `checkers.sh` because that should not have passed the build.